src/com/animation.js

```diff
@@ -173,13 +176,25 @@
 			return animation;
 		};
 
+		/**
+		 * Get the renderer instence used for this specific animation
+		 * @returns renderer instence
+		 */
+		var getRenderer = function () {
+			return renderer;
+		};
+
 		/**
 		 * Set the ctn of the animation and call render if everything is ready
 		 * @param {jQuery Element} scope the container of the animation
+		 * @param {function} cb callback when the render is done
 		 */
-		var init = function (scope) {
+		var init = function (scope, cb) {
```

Comment:
I would prefer to see a options parameter, with a `ready: ` property.
This way, it will be easier to add new options in the future.

Reply:
We can already pass an object at the creation of the component, set every options with `data-animation-*` like flickity. tbh I don't think we need another way to send config.

Reply:
All components are (or should be) able to get options from both the constructor and the init function.
This makes the signature uniform, and makes thing easy to learn.

Reply:
@fhamon ping!